legacy/briefinglocalizer/briefinglocalizerDlg.cpp

```diff
@@ -23,6 +23,10 @@
 #include "briefinglocalizer.h"
 #include "briefinglocalizerDlg.h"
 
+#ifdef __LINUX__
```

Comment:
This is dead code (`/legacy`) that we don't compile, I don't think this kind of attention is necessary

Reply:
If it didn't matter then it would be deleted. Until it's deleted I'm going to update it with the rest of the code.

Reply:
No problem, I was just making sure